docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,480 @@
+
+Together, you can deploy local test networks and run tests on them.
+
+## Getting Started
+
+To use AvalancheJS, you can clone the repo:
+
+```zsh
+$ git clone https://github.com/ava-labs/avalanchejs.git
+```
+
+or add it to an existing project:
+
+```zsh
+$ yarn add --dev avalanche
+```
+
+For this tutorial we will use a CLI to run the example scripts directly from an AvalancheJS directory.
+
+In order to send AVAX, you need to have some AVAX! You can use a pre-funded account on local network or get testnet AVAX from the [AVAX Test Faucet](https://faucet.avax.network), which is a free and easy way to get to play around with Avalanche. After getting comfortable with your code, you can run the code on Mainnet by making necessary changes.
+
+## Transferring AVAX using AvalancheJS
+
+The easiest way to transfer AVAX between chains is to use [AvalancheJS](https://github.com/ava-labs/AvalancheJS) which is a programmatic way to access and move AVAX.
+
+AvalancheJS allows you to create and sign transactions locally which is why it is the recommended way to transfer AVAX between chains. We are moving away from using AvalancheGo's keystore because it requires you to keep your keys on a full node which makes them a target for malicious hackers.
+
+You will find both a [Fuji Workflow](cross-chain-transfers.md#typical-fuji-workflow) and [Avalanche Network Runner Workflow](cross-chain-transfers.md#typical-avalanche-network-runner-workflow) in this tutorial.
+
+## Typical Avalanche Network Runner Workflow
+
+First confirm you have the latest AvalancheGo built.
+
+```text
+$ cd /path/to/avalanchego
+$ git fetch -p
+$ git checkout master
+$ ./scripts/build.sh
+```
+
+(Note that you can also [download pre-compiled AvalancheGo binaries](https://github.com/ava-labs/avalanchego/releases) rather than building from source.)
+
+Confirm you have Avalanche Network Runner installed by following the steps listed [here](create-a-local-test-network.md#installation)
+
+Start Avalanche Network Runner and run a script to start a new local network.
+
+### Step 1 - Start the server
+
+```text
+$ cd /path/to/Avalanche-Network-Runner
+$ avalanche-network-runner server \
+--log-level debug \
+--port=":8080" \
+--grpc-gateway-port=":8081"
+```
+
+### Step 2 - Start a New Avalanche Network with Five Nodes (a cluster)
+
+```bash
+# replace execPath with the path to AvalancheGo on your machine
+# e.g., ${HOME}/go/src/github.com/ava-labs/avalanchego/build/avalanchego
+$ AVALANCHEGO_EXEC_PATH="avalanchego"
+```
+
+```bash
+$ avalanche-network-runner control start \
+--log-level debug \
+--endpoint="0.0.0.0:8080" \
+--number-of-nodes=5 \
+--avalanchego-path ${AVALANCHEGO_EXEC_PATH}
+```
+
+Now you're running a local Avalanche network with 5 nodes.
+
+### Step 3 - Open AvalancheJS
+
+Open your AvalancheJS directory and select the **`examples`** folder to view the source code for the examples scripts.
```

Comment:
Please link the `example` to the github location `https://github.com/ava-labs/avalanchejs/tree/master/examples`